### DISCUSSION TITLE
Wildcard subdomain SSL certificates with Let's Encrypt and Bunny DNS | Rias.be

### DISCUSSION THREAD

**Discussion by giscus** [riasvdv/rias.be#549] [General]
# Wildcard subdomain SSL certificates with Let's Encrypt and Bunny DNS | Rias.be

Creating wildcard subdomain SSL certificates isn't that straightforward with Let's encrypt as a normal SSL certificate. As you need to prove you own the full domain before you can issue a certificate.

https://rias.be/blog/wildcard-subdomain-ssl-certificates-with-let-s-encrypt-and-bunny-dns

**Comment by esurov** [comment]
Thanks for the article. It would also be nice to cover the automatic certificates renewal.

**Comment by vishvendrasingh** [comment]
Any idea how to make it work with bunny pull zone, If I upload wildcard domain LetsEncrypt SSL I get error - SSL_ERROR_INTERNAL_ERROR_ALERT
